build.gradle

```diff
@@ -26,9 +25,48 @@ plugins {
     id 'net.researchgate.release' version '2.3.5'
 }
 
+subprojects {
+
+    group = "com.containersol.minimesos"
+    version = rootProject.version.toString()
+
+    repositories {
+        mavenCentral()
+        mavenLocal()
+        maven {
+            url "https://jitpack.io"
```

Comment:
But anyway. Nothing has been made worse than it already is by this change. Still it's something to keep an eye out for.